### ISSUE TITLE
Consider adding the Rosetta format library as a source

### ISSUE THREAD

**Issue by anjackson** [digipres/sentinel#20]
Available at: https://github.com/rosetta-format-library/RosettaFormatLibrary/releases/

> Rosetta FL is mostly mirroring PRONOM in format descriptions, with little differences. ... There are also things like associations between formats and metadata extractors, format classifications etc.

**Comment by thorsted** [comment]
We also have been associating formats with application within the Rosetta Format Library. @samalloing

**Comment by samalloing** [comment]
Yes in the format_library.xls there are different tabs for the different knowledge bases in Rosetta Format Library. The connection is made in the different tabs (see attachment for example for formats and applications)
![Capture](https://github.com/digipres/sentinel/assets/650574/508448cc-b6db-4893-b5d8-295f662db1ec)

I don't know what you see as scope of the technical registries and if the link between applications and formats is in scope. Or that you want to analyse what information registries store.

But anyway, you know how to find us or if you want to join the Digital Preservation Working Group of Rosetta as a guest that is all possible. Just talk to Jan and Micky. The PRONOM people sometimes join when it is relevant. If you want to talk about the Rosetta Format Library that is possible. So just think about it and let us know.

Sam
